src/pages/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en-En">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300&family=Montserrat:wght@500;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/style.css">
+    <script src="https://kit.fontawesome.com/52ee460aa5.js" crossorigin="anonymous"></script>
+    <script type="module" src="../script/app.js"></script>
+    <title>Ackroyd</title>
+</head>
+
+<body>
+    <nav class="nav">
+        <h2 class="logo">Ackroyd</h2>
+        <ul>
+            <li><a href="./index.html" class="nav__link">Home</a></li>
+            <li><a href="./index1.html" class="nav__link" id="diffrent_color">Moon</a></li>
```

Comment:
You use `id` for styling purposes, add different CSS class instead. It's good as a convention and & it has the same specificity.